pkg/cluster/controller.go

```diff
@@ -37,7 +37,6 @@ func runController(ctx context.Context, dataDir string) error {
 	args := []string{
 		"--bind-address=0.0.0.0",
 		"--secure-port=11252",
-		"--port=0", // Don't serve insecure
```

Comment:
This flag is throwing a warning. I can't even find it in the 1.18 docs.

Reply:
@divolgin I _believe_ you found this one and solved a problem with it?